CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+## 1.1.3 (2023-12-04)
+
+Full Changelog: [v1.1.3]()
+
+### Bug Fixes
+
+- **sdk:** Updated `add_prompt` to rename prompt name if a different name is passed in ([a29d5f87](https://github.com/lastmile-ai/aiconfig/pull/303/commits/a29d5f87198b42a46335c4f20820f02b7c3b1abf))
+- **typescript-sdk:** Updated Metadata Field to be optional ([cb5fdc5](https://github.com/lastmile-ai/aiconfig/pull/320/commits/cb5fdc51a4c6797352c8c6e8b6eb6deb9fdde32b))
+
+### Feature
+
+- **python-sdk:** Extension for HF Text Generation with transformers ([222bf6e](https://github.com/lastmile-ai/aiconfig/pull/348/commits/222bf6edf5c615b3b9b5f58b7fd5c829b2884d1b))
+- **python-sdk:** DALL-E Model Parser ([4753f21](https://github.com/openai/openai-python/commit/71a13d0c70d105b2b97720c72a1003b942cda2ae))
+- **python-sdk:** Improved OpenAI Introspection wrapper ([143c3dd](https://github.com/lastmile-ai/aiconfig/pull/364/commits/143c3dd72b4d46e9c4f38a89492dc8404079e3d6))
+
+### Chores
```

Comment:
Rename to "Better Engineering"

Reply:
Maybe we can be more descriptive, e.g. devX enhancements or code quality improvements?

Reply:
I feel like Better Engineering captures those. I'm hoping the structure of the changelog stays the same over time.

Reply:
**Chores** was a direct copy from OpenAI, but we can change it to something else. Better Engineering is very Facebook, but it comes with connotations.

Reply:
Updated to Better Engineering, unless any strong suggestions for a different descriptor